usefulUtilities/modelJSONCleanup/README.md

```diff
@@ -0,0 +1,18 @@
+# Description
+
+Node script that can be used to help cleanup models.json
+
+# Use
+
+- Get an archive of the domain you would like
+- Extract the zip
+- Extract the models gzip file
+- Duplicate the generic js filter file for the specific domain
```

Comment:
what does this step mean?

Reply:
Going to remove now that you mentioned the config file.  Definitely, a better way to go.